### PR TITLE
Update to edition-arica of the Eiffel protocol

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

fixes: #67 

### Description of the Change

Updated to the latest schemas

### Alternate Designs

N/A

### Benefits

N/A

### Possible Drawbacks

N/A

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias Persson <tobias.persson@axis.com>

### CODE REVIEW

**Comment by t-persson** [comment]
I'll try to break out the workflow changes and create a separate PR for those (including fixing the warnings and errors) during the day or tomorrow

File: src/eiffel_graphql_api/graphql/schemas/events/json_schemas/EiffelActivityStartedEvent.json [hunk 1]
```diff
@@ -52,24 +56,69 @@
...
+                  "position": {
+                    "type": "integer"
+                  }
+                },
+                "additionalProperties": false,
+                "required": [
+                  "sequenceName",
+                  "position"
+                ]
+              }
```
Comment (magnusbaeck) [inline src/eiffel_graphql_api/graphql/schemas/events/json_schemas/EiffelActivityStartedEvent.json:112]: This diff is really weird. It looks like the ActS and ActT schemas currently on master are very out of date.

**Review by magnusbaeck — CHANGES_REQUESTED** [review]
You need to update the [link types table](https://github.com/eiffel-community/eiffel-graphql-api/blob/ab63a8470c79447b0772055eef4c8e218e3c3383/src/eiffel_graphql_api/graphql/schemas/links/utils.py) too, right? RUNTIME_ENVIRONMENT and PRECURSOR are missing (maybe more).

**Review by fredjn — APPROVED** [review]

File: src/eiffel_graphql_api/graphql/schemas/events/json_schemas/EiffelActivityStartedEvent.json (hunk 1 above)
Comment (t-persson) [reply src/eiffel_graphql_api/graphql/schemas/events/json_schemas/EiffelActivityStartedEvent.json:112]: They are, yes :) Been bad at updating

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 sphinx<2
 pyscaffold==2.5.10
+mistune==0.8.4
```
Comment (magnusbaeck) [inline docs/requirements.txt:3]: Where and how is this dependency used?

**Review by magnusbaeck — APPROVED** [review]

File: docs/requirements.txt (hunk 2 above)
Comment (t-persson) [reply docs/requirements.txt:3]: It is used by m2r (the line below) and m2r does not set version and the project is set to read-only. This meant that m2r wanted to install a version of mistune that was not backwards compatible thus breaking m2r for us.

Comment (t-persson) [reply docs/requirements.txt:3]: I want to remove m2r when we are updating to support newer python versions

**Comment by t-persson** [comment]
Added a commit with `Older gevent version is not compatible with greenlet` as well as a rebase against master

**Comment by magnusbaeck** [comment]
Still LGTM :-)

**Review by fredjn — APPROVED** [review]

**Pull request merged**